platform/packages/finance/src/price/base/mod.rs

```diff
@@ -227,6 +227,19 @@ mod test_invariant {
         }
     }
 
+    #[test]
+    #[should_panic = "Failed to deserialize"]
```

Comment:
Next time leave tests fail until fixing the code. This is the way not to forget it.